crates/egui/src/widgets/text_edit/state.rs

```diff
@@ -44,6 +44,10 @@ pub struct TextEditState {
     #[cfg_attr(feature = "serde", serde(skip))]
     pub(crate) has_ime: bool,
 
+    // target ID for IME candidate.
+    #[cfg_attr(feature = "serde", serde(skip))]
+    pub(crate) ime_target_id: Id,
```

Comment:
Won't this always be the same `Id` as the key used to find `TextEditState`?

Reply:
Yes, it only stores TextEdit ID.                                                      
It's currently working fine without any problems.                                                                                      
If something is wrong or needs to be fixed, please fix it or let me know.

Reply:
That the code is working fine is not enough. I need to maintain this, and so I need to understand it. Storing the `Id` here makes no sense to me, since it is already the key.